en/config_vtol/vtol_without_airspeed_sensor.md

```diff
@@ -32,49 +34,45 @@ The flight should be performed at a speed that would be sufficient to fly in hig
 After the reference flight download the log and use [FlightPlot](../log/flight_log_analysis.md#flightplot) (or another analysis tool) to examine the log.
 Plot the altitude (`GPOS.Alt`), thrust (`ATC1.Thrust`), groundspeed (Expression: `sqrt(GPS.VelN\^2 + GPS.VelE\^2)`), pitch (`ATT.Pitch`) and roll (`AT.Roll`).
 
-Examine the throttle level (thrust) when the vehicle is level (no or little pitch and roll), 
-during the ascend (increasing altitude) and when the vehicle is banking (more roll). 
-The initial value to use as cruise speed should be the highest thrust applied during a roll or ascend, 
-the thrust during level flight should be considered the minimum value if you decide to further tune down your speed.
+Examine the throttle level (thrust) when the vehicle is level (no or little pitch and roll), during the ascend (increasing altitude) and when the vehicle is banking (more roll).

```

Comment:
FYI only I tend to break on sentences. It's a preference that helps me review the flow of docs when editing, but I also think most comments apply to a sentence, so it's not a bad point to break.